content/applications/websites/livechat/ratings.rst

```diff
@@ -2,44 +2,138 @@
+Editor`.
+
+To publish the `livechat` webpage, return to the list of webpages by navigating to
+:menuselection:`Site --> Pages`. Click the checkbox to the left of `livechat` in the list of pages
+to select the page. Then, click the checkbox under the column labeled :guilabel:`Is Published`. Then
+click :guilabel:`Save`.
+
+.. image:: ratings/live-chat-is-published.png
    :align: center
-   :alt: View of a chat window from an operator’s side highlighting a rating for Odoo Live Chat
+   :alt: View of the list of pages for a website with the 'is published' box emphasized.
+
+Once the page has been added to the site, ratings are set to be published by default. However,
+individual ratings can be manually selected to be hidden from the public. The rating will still be
+included in internal reports, and can still be viewed by internal teams. However, public website
+visitors and portal users will not have access.
+
+See :ref:`Hide individual ratings <livechat/overview/hide-ratings>` for more information.
+
+Customer ratings report
+=======================
 
-And under :menuselection:`Report --> Customer Ratings`.
+The :guilabel:`Customer Ratings` report (:menuselection:`Live Chat --> Report --> Customer Ratings`)
+displays an overview of the ratings received on individual support tickets, as well as any
+additional comments submitted with the rating.
 
-.. image:: ratings/customer_ratings.png
+.. image:: ratings/live-chat-ratings-report.png
    :align: center
-   :alt: View of the customer ratings page in Odoo Live Chat
+   :alt: View of the customer ratings report in Odoo Live Chat.
+
+Click on an individual rating to see additional details about the conversation, and the rating.
+
+.. _livechat/overview/hide-ratings:
+
+Hide individual ratings
+-----------------------
 
-Make the rating public
-======================
+Ratings are set to be published by default. However, individual ratings can be manually selected to
+be hidden from the public. The rating will still be included in internal reports, and can still be
+viewed by internal teams. However, public website visitors and portal users will not have access.
 
-Go to :menuselection:`Website --> Configuration --> Settings`, access your channel form, click on
-*Go to Website* and on *Unpublished*, to publish the rating of that channel on your website.
+To hide a rating, go to :menuselection:`Live Chat --> Reports --> Customer Ratings`. Click on the
+kanban card for a rating to be hidden. On the rating's detail page, check the box labeled
+:guilabel:`Visible Internally Only`.
 
-.. image:: ratings/publish_ratings.png
+.. image:: ratings/live-chat-ratings-visible-internally.png
    :align: center
-   :alt: View of the public ratings in the website for Odoo Live Chat
+   :alt: View of an individual rating's detail page with the visible internally setting checked.
 
 .. seealso::
+   - :doc:`/applications/websites/livechat`
    - :doc:`responses`
+   - :doc:`/applications/websites/website`
```

Comment:
Just a general comment, so I added it on this last line! It might be good to add an explanation of how to organize the feedback in kanban columns, sorted by rating. I think that might be helpful than the default view. It might be good for them to know. This might be covered in a reporting doc if that's in the works, but if not, I like to add things like that which could be very helpful, esp. for a first time. They are probably asked to aggregate the #'s, etc, and that might be easy and add value.